development/conf.py

```diff
@@ -33,7 +33,8 @@
     'sensio.sphinx.configurationblock',
     'sensio.sphinx.phpcode',
     'sensio.sphinx.bestpractice',
-    'sphinxcontrib.phpdomain'
+    'sphinxcontrib.phpdomain',
+    'sphinx_multiversion'
```

Comment:
do we need `sphinx_rtd_theme` in here as well?

Reply:
Don't think so as that's a theme and not an extension?

Reply:
I only asked cuz I saw them doing it here

https://github.com/mixxxdj/manual/blob/2.2/source/conf.py#L35